spec/fixtures/acceptance-credentials.conf

```diff
@@ -0,0 +1,4 @@
+address: bk5xpslulx2jcsd.delivery.puppetlabs.net
```

Comment:
Can this and the `acceptance-device.conf` be removed from the commit?

Reply:
rm'd